src/shell.js

```diff
@@ -183,11 +183,9 @@ var read_,
 function logExceptionOnExit(e) {
   if (e instanceof ExitStatus) return;
   let toLog = e;
-#if EXCEPTION_STACK_TRACES
```

Comment:
This was added in #18642 but now I'm not sure why. Adding this condition suppresses even JS `Error` traces not generated from Wasm code, which was not what I intended.